### ISSUE TITLE
Mixing freezes on access denied problem casually

### ISSUE THREAD

**Issue by krychla1** [akhavr/electrum-dash#290]
Dear electrum support,
the private send mixing gets stuck time to time (every few hours) on access denied problem. Happens to us on multiple different PCs. 

```
08/23/21 11:15:08 error starting mixing: [WinError 5] Access is denied: '###CENSORED###\\electrum-dash\\wallets\\wallet_3.backup.tmp.21224' -> '###CENSORED###\\electrum-dash\\wallets\\wallet_3.backup'
```

Might be kind of conflicts on parallel writes.

The question is how to prevent this and(or) how to handle it. Currently we have to check the logs repeatedly and manually restart the mixing if it happens. 

Thanks

**Comment by akhavr** [comment]
Are you sure your `\\electrum-dash\\wallets` folder is writeable by the user, you're using to run `electrum-dash`?  The message says that it can't create a backup wallet file.

**Comment by krychla1** [comment]
Yes it is, otherwise it wouldn't mix and write data to the wallet file at all.

**Comment by akhavr** [comment]
Strange.  Will try to reproduce in a VM,

**Comment by zebra-lucky** [comment]
After repeated reading of issue, realized the problem can be related to antivirus software.

If scan of wallets folder is in the progress, some file operations can be blocked, and this
error can occur.

**Comment by zebra-lucky** [comment]
I'll try to reproduce the problem with mcAfee and defender.

**Comment by zebra-lucky** [comment]
Reproduced with McAfee.

During scan of wallets folder and wallet save operation
at the same time, `[WinError 5] Access is denied` error is raised.

**Comment by krychla1** [comment]
Dear devs,
this errors still occurs (4.1.5.0). The conflict itself is not a big deal from the user perspective, the main problem is that the mixing stops and has to be manually restarted. 

Perhaps you should check the error handling on the conflict.

Thanks.

**Comment by zebra-lucky** [comment]
Some instructions for McAfee:
https://download.mcafee.com/products/webhelp/5/affid/756/1033/GUID-BA648B78-244A-47F1-8AD6-4A57BA8955BC.html

**Comment by krychla1** [comment]
Thank you zebra for tips. Unfortunately I am not privileged to adjust antivirus settings on the machines.
